src/codegen/templates/_grpc_interpreter.py.mako

```diff
@@ -0,0 +1,66 @@
+<%
+    from codegen.utilities.interpreter_helpers import get_interpreter_functions, get_grpc_interpreter_call_params, get_interpreter_params, get_interpreter_parameter_signature, get_output_params, get_response_parameters, get_compound_parameter, create_compound_parameter_request, get_input_arguments_for_compound_params
+    from codegen.utilities.function_helpers import order_function_parameters_by_optional
+    from codegen.utilities.text_wrappers import wrap, docstring_wrap
+    from codegen.utilities.helpers import snake_to_pascal
+    functions = get_interpreter_functions(data)
+%>\
+# Do not edit this file; it was automatically generated.
+
+import grpc
+
+from . import errors as errors
+from nidaqmx._base_interpreter import BaseInterpreter
+from nidaqmx._stubs import nidaqmx_pb2 as grpc_types
+from nidaqmx._stubs import nidaqmx_pb2_grpc as nidaqmx_grpc
+from nidaqmx._stubs import session_pb2 as session_grpc_types
+
+
+
+class GrpcStubInterpreter(BaseInterpreter):
+    '''Interpreter for interacting with a gRPC Stub class'''
+
+    def __init__(self, grpc_options):
```

Comment:
On the topic of `__slots__`, let's specify `__slots__` for the interpreters. Related: https://github.com/ni/nidaqmx-python/pull/299#discussion_r1175785977

```suggestion
    '''Interpreter for interacting with a gRPC Stub class'''
    __slots__ = ['_grpc_options', '_client']

    def __init__(self, grpc_options):
```